src/rtc/defaults/rtc_s1.yaml

```diff
@@ -1,147 +1,183 @@
 runconfig:
-  name: rtc_s1_workflow_default
-
-  groups:
-      pge_name_group:
-          pge_name: RTC_S1_PGE
-
-      input_file_group:
-          # Required. List of SAFE files (min=1)
-          safe_file_path:
-          # Required. List of orbit (EOF) files (min=1)
-          orbit_file_path:
-          # Required. The unique burst ID to process
-          burst_id:
-
-      dynamic_ancillary_file_group:
-          # Digital elevation model
-          dem_file:
-
-      product_path_group:
-          # Directory where PGE will place results
-          product_path:
-          # Directory where SAS writes temporary data
-          scratch_path:
-
-          # If option `mosaic_bursts` is not set, output files are saved to:
-          #     {output_dir}/{burst_id}/{product_id}{suffix}.{ext}
-          # If option `mosaic_bursts` is set, output files are saved to:
-          #     {output_dir}/{product_id}{suffix}.{ext}
-          # If the field `product_id`` is left empty, the prefix "rtc_product"
-          # will be used instead.
-          # `suffix` is only used when there are multiple output files.
-          # `ext` is determined by geocoding_options.output_format.
-          output_dir:
-          product_id:
-          mosaic_bursts: False
-
-          # Format of output file
-          output_format: NETCDF
-
-
-      primary_executable:
-          product_type: RTC_S1
-
-      processing:
-          polarization:
-          geo2rdr:
-              threshold: 1.0e-8
-              numiter: 25
-
-          dem_interpolation_method: biquintic
-
-          # Apply absolute radiometric correction
-          apply_absolute_radiometric_correction: True
+    name: rtc_s1_workflow_default
+
+    groups:
+        pge_name_group:
+            pge_name: RTC_S1_PGE
+
+        input_file_group:
+            # Required. List of SAFE files (min=1)
+            safe_file_path:
+            # Required. List of orbit (EOF) files (min=1)
+            orbit_file_path:
+            # Required. The unique burst ID to process
+            burst_id:
+
+        dynamic_ancillary_file_group:
+            # Digital elevation model
+            dem_file:
+
+            # Digital elevation model source description
+            dem_description:
+
+            # burst database sqlite file
+            burst_database_file:
```

Comment:
Thanks for adding the database. However the database is supposed to be static ancillary file and not dynamic.

Reply:
Got it, will update the runconfig and code accordingly. Thanks.